src/scryptTypes.ts

```diff
@@ -29,7 +28,10 @@ export class ScryptType {
   }
 
   get finalType(): string {
-    return this._finalType;
+    const alias: AliasEntity[] =  Object.getPrototypeOf(this).constructor.alias;
```

Comment:
Better do this in constructor function or outside of the class.

Reply:
refactor: using _typeResolver
18e4b4264b4ac49eb31218dad09ba0283c7cb83b